test/FunctionalTests/Infrastructure/ApplicationTestFixture.cs

```diff
@@ -67,56 +61,29 @@ public static DeploymentParameters GetDeploymentParameters(string applicationPat
             return deploymentParameters;
         }
 
-        public virtual ILoggerFactory CreateLoggerFactory()
-        {
-            return new LoggerFactory().AddConsole();
-        }
-
         public void Dispose()
         {
-        }
-
-        private static void TryDeleteDirectory(string directory)
-        {
-            try
+            if (_deploymentTask?.Status == TaskStatus.RanToCompletion)
             {
-                Directory.Delete(directory, recursive: true);
```

Comment:
We used this to delete the restore directory in the past but this was left over from since we changed how it's done.